### PR TITLE
Dz patch 1

### PR DESCRIPTION
**Pull request by daviezwei**
Completely new to this github stuff so let me know if I need to be doing something differently.

### CODE REVIEW

**Comment by daviezwei** [comment]
Thanks. I wasn't sure if they were too much/too wordy but if they're good
then yay!

And sure, I suppose I can take a look at the lusty lass ones next.

**Comment by JJM152** [comment]
Great, that'd be a big help. I really did a worse than usual job on those just to get them into the game.

If you get bored with that or want something easy to do, I have a lot of empty passages still that just plain need descriptions. 

You could also add some more clothing to the game or TF items. Both are pretty easy to work out how to do.

**Comment by daviezwei** [comment]
Awesome. Thanks.

I wanted to ask if there was a way currently to do if/else checks on
clothes/skills/etc in these passages? I'm not a programmer by any stretch
so a lot of this is way over my head, but I'd like to include references to
different things if I could. For example, if the protagonist is locked in
chastity, barefoot, skilled in a particular sex act, etc.

On Fri, May 25, 2018 at 11:21 AM, Mystery Game Dev <notifications@github.com

**Comment by JJM152** [comment]
In passages,yes - I use it sparingly myself, but it's easy to do. Just inline something like this:

`<<if $Player.GetStat("BODY", 'Bust') >= 60>>
Your massive boobs stop you from being able to crawl down the tunnel.
<<else>>
You shimmy and shake your way down the narrow passage.
<</if>>`

This is standard sugarcube notation. You can also print out values from code easily by doing things like:

`<<= $Player.SlaveName >>`

There are plenty of things baked into the game on the $Player object for you to check. Just have a peak at the file /src/000-SCRIPT_OBJ/Player.js - you can access all the players data there during run time.

There are also some helper function in the App.PR namespace (Helper.js) that will print out an adjective or description based on a players body part statistic, for example

`<<= App.PR.pBust( $Player) >>` - will print out the mirror description of the body part
`<<= App.PR.pBust( $Player, true) >>` - will print out an adjective based on the bust size

They exist for most things that you can describe about the character. If you think one is missing let me know and I'll hack it in.

Cheers
